README.md

```diff
@@ -132,11 +133,11 @@ let receiptIdChunks = expo.chunkPushNotificationReceiptIds(receiptIds);
 
 The source code is in the `src/` directory and babel is used to turn it into ES5 that goes in the `build/` directory.
 
-To build, `npm run build`.
+To build, `yarn build`.
```

Comment:
When I run `npm run build` locally I don't run into any issues. What problem did you have?

Reply:
There is no problem. 
In this project, I noticed that there is a yarn.lock file and no package-lock.json file, so I assumed you want contributors to use Yarn.

Reply:
I see. Because this project is rather simple, I don't think it matters either way. You can run the scripts however you'd like and I don't think the differences between Yarn and npm will matter here.

Reply:
I got it.
So Should I revert it or not?

Reply:
Let's keep your changes, it's slightly better if people use Yarn.

Reply:
Thanks bro.